src/components/Heroes/TestTypescriptReact.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const TestTypescriptReact = () => (
+  <div>Test</div>
+);
+
+export default TestTypescriptReact;
```

Comment:
Should we remove this file?

Reply:
Yes was just testing all of the linting with this.